components/NavBar.tsx

```diff
@@ -33,22 +89,44 @@ const styles: Record<string, React.CSSProperties> = {
   container: {
     display: 'flex',
     alignItems: 'center',
-    backgroundColor: '#c7f1ff',
+    backgroundColor: 'white',
     top: 0,
     justifyContent: 'space-between',
     padding: '16px 16px',
   },
+  leftNav: {
+    display: 'flex',
+    alignItems: 'center',
+  },
   logoContainer: {
     display: 'flex',
     alignItems: 'center',
+    paddingRight: '32px',
   },
   logoFont: {
     padding: '0px 0px 10px 20px',
     fontSize: '30px',
   },
-  linkContainer: {
+  leftLinks: {
+    display: 'flex',
+    gap: '16px',
+  },
+  rightLinks: {
     display: 'flex',
     gap: '16px',
+    alignItems: 'center',
+  },
+  primaryButton: {
+    backgroundColor: '#19303D', // charcoal
+    color: 'white',
+    padding: '8px 22px',
+    borderRadius: '3px',
+  },
+  secondaryButton: {
```

Comment:
It looks like in mocks this should be a ghost button

Reply:
Was a text button. Not sure why I didn't style this one after I did the other two 😂